src/core_modules/capture-core/components/DataEntry/dataEntryField/internal/DataEntryField.component.tsx

```diff
@@ -97,10 +54,10 @@ class DataEntryFieldPlain extends React.Component<Props> {
             componentProps,
             ...passOnProps
         } = this.props;
-        const { isValid, type, ...passOnValueMeta } = valueMeta;
+        const { isValid, type, ...passOnValueMeta } = valueMeta as ValueMetaInput;
         return (
             <div
-                ref={(gotoInstance) => { this.gotoInstance = gotoInstance; }}
+                ref={(gotoInstance) => { this.gotoInstance = gotoInstance || undefined; }}
```

Comment:
Remove `|| undefined` and fix the error by adding `gotoInstance?: HTMLDivElement | null;` on line 9